packages/integration-tests/projects/suite-web/tests/suite/new-account-message.test.ts

```diff
@@ -0,0 +1,42 @@
+// @group:suite
+// @retry=2
+
+describe('New accounts', () => {
+        beforeEach(() => {
+             // Launches Trezor emulator
+            cy.task('startEmu', { wipe: true });
+            // Initializes Trezor (random seed)
+            cy.task('setupEmu', { needs_backup: false });
+            cy.task('startBridge');
+            cy.viewport(1024, 768).resetDb();
+            cy.prefixedVisit('/');
+            cy.passThroughInitialRun();
+            cy.discoveryShouldFinish();
+        });
+    
+
+    it('Goes to accounts and verifies that the "New default Accounts natively.." is displayed:', () => {
+        cy.prefixedVisit('/accounts/');
+        cy.getTestElement('@accounts/empty-account/receive');
+        cy.getTestElement('@accounts/empty-account/default-native-account/close');
+        cy.contains('New default Accounts natively in Trezor Suite!');
+        // Clicks on Got it!
+        cy.contains('New default Accounts natively in Trezor Suite!');
```

Comment:
Any special reason for having these two same lines?

Reply:
duplicity, deleted with cb0a3c4762e4b4db44b9de5a6c79d524fdc8e310